### PR TITLE
Documented all currently available nodes for Llamachain & Llamaindex

### PR DESCRIPTION
**Pull request by toi500**
**Added around 100 new documented nodes (LangChain & Llamaindex):** 

I've documented all nodes relevant to the Llamachain and Llamaindex frameworks, leaving previously documented nodes untouched. 

**Each new node entry includes the following information:**

1. Title
2. System Description (matches in-app description)
3. Node Capture (image/screenshot)
4. Caption
5. Call to Action (CTA): A clear CTA informing users that the new doc section is a work in progress and welcomes contributions.

**Example:**

image

This documentation should significantly enhance the user experience by providing a clear understanding of the nodes within these frameworks.

**Here is the final resul:** 
https://toi500.gitbook.io/flowise-docs/integrations

### CODE REVIEW

**Comment by toi500** [comment]
@HenryHengZJ can you advise or comment about this PR? I have put bit of effort here and user like it:

image

**Comment by HenryHengZJ** [comment]
@toi500 words cant express how grateful I am to all the docs update, really really appreciated!!

there are some merge conflicts on the images, would you be able to resolve those? let me know if you need any help!

**Review by toi500 — COMMENTED** [review]
re-upload imgs

**Comment by toi500** [comment]
@HenryHengZJ I think is done.

File: integrations/README.md [hunk 1]
```diff
@@ -2,13 +2,37 @@
...
-* [Chat Models](langchain/chat-models/)
-* [Embeddings](langchain/embeddings/)
-* [Memory](langchain/memory/)
-* [Text Splitters](langchain/text-splitters/)
-* [Tools](langchain/tools/)
-* [Vector Stores](langchain/vector-stores/)
-* [External Integrations](3rd-party-platform-integration/)
+### LangChain Nodes
+
+* [Agents](https://toi500.gitbook.io/flowise-docs/integrations/langchain/agents)
```
Comment (HenryHengZJ) [inline integrations/README.md:7]: we need to update the link

File: integrations/llamaindex/vector-stores/queryengine-tool.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (HenryHengZJ) [inline integrations/llamaindex/vector-stores/queryengine-tool.md:1]: we should have pinecone and simplestore nodes

Comment (HenryHengZJ) [reply integrations/llamaindex/vector-stores/queryengine-tool.md:1]: After updates, we need to update SUMMARY.md as well

Comment (toi500) [reply integrations/llamaindex/vector-stores/queryengine-tool.md:1]: done

File: integrations/README.md (hunk 1 above)
Comment (toi500) [reply integrations/README.md]: done

**Comment by toi500** [comment]
@HenryHengZJ missing nodes for LlamaIndex added and links and summary.md updated.

**Review by HenryHengZJ — APPROVED** [review]

**Comment by HenryHengZJ** [comment]
thank you!!

**Comment by toi500** [comment]
@HenryHengZJ I think this is a good foundation to build on.

Next logical step would be to provide a short conceptual intro (and add links) to the sections:
image

Let me know if you think it is a good idea so I start. 

I could use and refrase the info that LangChain has:
https://js.langchain.com/v0.2/docs/concepts

**Pull request merged**